src-literatetests/30-tests-context-free.blt

```diff
@@ -796,22 +796,22 @@ import Prelude as X
     )
 
 #test long-module-name-simple
-import TestJustShortEnoughModuleNameLikeThisOne ()
-import TestJustAbitToLongModuleNameLikeThisOneIs ()
 import MoreThanSufficientlyLongModuleNameWithSome
-  (items, that, will, not, fit, inA, compact, layout)
+  (compact, fit, inA, items, layout, not, that, will)
```

Comment:
😆 These are funny now that they're sorted.

Reply:
If you can think of an alphabetical sentence with the same meaning we should use that!